rgw/admin/bucket_test.go

```diff
@@ -26,6 +32,45 @@ func (suite *RadosGWTestSuite) TestBucket() {
 		assert.True(suite.T(), errors.Is(err, ErrNoSuchBucket), err)
 	})
 
+	suite.T().Run("info existing bucket", func(t *testing.T) {
+		_, err := co.GetBucketInfo(context.Background(), Bucket{Bucket: suite.bucketTestName})
+		assert.NoError(suite.T(), err)
+	})
+
+	suite.T().Run("create test user for bucket/quota test", func(t *testing.T) {
+		user, err := co.CreateUser(context.Background(), User{ID: "quota-test", DisplayName: "This is user to test quotas", Email: "quota@example.com"})
+		assert.NoError(suite.T(), err)
+		assert.Equal(suite.T(), "quota@example.com", user.Email)
+	})
+
+	suite.T().Run("set user quota", func(t *testing.T) {
+		maxObjects := int64(100)
+		err := co.SetQuota(context.Background(), Quota{QuotaType: "user", UID: "quota-test", MaxObjects: &maxObjects})
```

Comment:
Please enable quota in this request
```suggestion
              quotaEnable := true
		err := co.SetQuota(context.Background(), Quota{QuotaType: "user", UID: "quota-test", MaxObjects: &maxObjects, Enabled: &quotaEnable})
```